new_matrixfit.Rmd

```diff
@@ -33,39 +37,236 @@ uw2 <- uwerr.cf(corr2)
 save(uw2, file = 'uw2.Rdata')
 uw3 <- uwerr.cf(corr3)
 save(uw3, file = 'uw3.Rdata')
+
+samplecf_boot1 <- bootstrap.cf(corr1, boot.R = boot.R, boot.l = boot.l, seed = seed)
+shifted_corr1 <- takeTimeDiff.cf(samplecf_boot1) # Calculate the shifted correlator
+
+ samplecf_boot2 <- bootstrap.cf(corr2, boot.R = boot.R, boot.l = boot.l, seed = seed)
+ shifted_corr2 <- takeTimeDiff.cf(samplecf_boot2) # Calculate the shifted correlator
+ 
+ samplecf_boot3 <- bootstrap.cf(corr3, boot.R = boot.R, boot.l = boot.l, seed = seed)
+ shifted_corr3 <- takeTimeDiff.cf(samplecf_boot3) # Calculate the shifted correlator
+ 
+ samplecf_boot4 <- bootstrap.cf(corr4, boot.R = boot.R, boot.l = boot.l, seed = seed)
+ shifted_corr4 <- takeTimeDiff.cf(samplecf_boot4) # Calculate the shifted correlator
+ 
+ samplecf_boot5 <- bootstrap.cf(corr5, boot.R = boot.R, boot.l = boot.l, seed = seed)
+ shifted_corr5 <- takeTimeDiff.cf(samplecf_boot5) # Calculate the shifted correlator
 ```
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
 ```{r}
-samplecf_boot1 <- bootstrap.cf(corr1, boot.R = boot.R, boot.l = boot.l, seed = seed)
-shifted_corr1 <- takeTimeDiff.cf(samplecf_boot1) # Calculate the shifted correlator
+## MASTER ANALYSIS:
+
+# c1_T48L10 <- new_matrixfit(samplecf_boot1, 1, 23, fit.method = 'lm', model = 'single', sym.vec = 'cosh', useCov = TRUE, par.guess = c(0.22,0.17))
```

Comment:
Lots of commented out stuff here, why is that?

Reply:
Basically this Rmd file is needless. I just used it to read in data. Because I tried different things, I commented out some stuff.

Reply:
so, please, clean up!

Reply:
Now the Rmd file is removed.